### ISSUE TITLE
OIDC/openid-connect broken

### ISSUE THREAD

**Issue by benshalev849** [PowerDNS-Admin/PowerDNS-Admin#1498]
### PDA version

0.4.0 (dev branch)

### Python version

3.10

### Steps to Reproduce

1. set all needed URLs but JWKS and metadata URL.
2. Log in to the application.

other way:
1. Upgrade from older version (legacy)
2. Login to SSO.

### Expected Behavior

Log in normally with OIDC, like it used to work.

### Observed Behavior

First without setting JWKS URL i get:
"RuntimeError: Missing "jwks_uri" in metadata"

After setting that i still get an error, so i set metadata URL and still get error.
"RuntimeError: Missing "jwks_uri" in metadata"

Now i set metadata URL and i get bad request to the metadata URL, with my metadata URL being a `/userinfo` endpoint.

I think there should be an option or that these URL's won't be mandatory since OIDC worked up untill now, saw some other issues relating this but did not find any that solved the problems for us.
TY :)

Same probelm as:
https://github.com/PowerDNS-Admin/PowerDNS-Admin/issues/1445

And:
https://github.com/PowerDNS-Admin/PowerDNS-Admin/issues/1489

That still occurs in normal OIDC :)

**Comment by AzorianMatt** [comment]
@benshalev849 please read the contribution guide. Duplicate issues are against the rules.

**Comment by AzorianMatt** [comment]
Take a look at [this issue](https://github.com/PowerDNS-Admin/PowerDNS-Admin/issues/1458) @benshalev849.

**Comment by benshalev849** [comment]
@AzorianMatt 

Ok, found out the metadata URL was wrong by asking the AUTH team in our org.

I think the documentation for the OIDC migration to 4.0 is something that should be added in upgrade notes.

**Comment by AzorianMatt** [comment]
@benshalev849 in all fairness, this wasn't an anticipated requirement change for the release. It's the result of an oversight on part of at least myself.

I just pushed some changes into the `dev` Branch this morning that should make the metadata URL unused if empty. Presumably, this should make not setting a value for the setting the same as how it worked before it was added.

If you're running this with our docker image, give the `dev` tag a try for me and let me know if it corrects the issue without the use of a metadata URL.

Thanks!
